website/pages/en/deploying/deploying-a-subgraph-to-studio.mdx

```diff
@@ -1,17 +1,36 @@
 ---
-title: Deploying a Subgraph to Subgraph Studio
+title: Deploy Using Subgraph Studio
 ---
 
-These are the steps to deploy your subgraph to Subgraph Studio:
+Learn how to deploy your subgraph in Subgraph Studio.
 
-- Install The Graph CLI (with either yarn or npm)
-- Create your Subgraph in Subgraph Studio
-- Authenticate your account from the CLI
-- Deploying a Subgraph to Subgraph Studio
+> Note: When you deploy a subgraph, you push it to Subgraph Studio, where you'll be able to stage it. It's important to remember that deploying is not the same as publishing. When you publish a subgraph, you're publishing it on-chain.
```

Comment:
Is "stage" the term we want to use? I feel like "test" is more accessible.

Reply:
-This change was in response to feedback from devs